cmd/createVolume.go

```diff
@@ -107,6 +107,10 @@ func createVolumeExec(cmd *cobra.Command, args []string) error {
 	if cvOpts.req.Spec.HaLevel == 0 {
 		cvOpts.req.Spec.HaLevel = 1
 	}
+	// Set ext4 as default value for filesystem.
```

Comment:
We need a --fs = ext4, none, default to ext4

Reply:
Updated the patch with the request change. More details in the description.